File: SqlOrganize/Sql/EntityRelation.cs
namespace SqlOrganize.Sql
{
    public class EntityRelation
    {
        public string fieldName { get; set; }
        public string refEntityName { get; set; }
        public string refFieldName { get; set; } = "id";
        public string? parentId { get; set; }
    }
}

File: src/SqlOrganize/Sql/EntityRelation.php
<?php

namespace SqlOrganize\Sql;

class EntityRelation
{
    public string $fieldName;
    public string $refEntityName;
    public string $refFieldName = 'id';
    public ?string $parentId = null;

    public static function getInstance($fieldName, $refEntityName, $refFieldName = "id"): EntityRelation {
        $et = new EntityRelation();
        $et->fieldName = $fieldName;
        $et->refEntityName = $refEntityName;
        $et->refFieldName = $refFieldName;
        return $et;
    }
}
